Framework/Kernel/src/Matrix.cpp

```diff
@@ -110,32 +114,32 @@ Matrix<T>::Matrix(const std::vector<T> &A, const std::vector<T> &B)
   @param B :: Row vector to multiply
 */
 {
-  // Note:: nx,ny zeroed so setMem always works
+  // Note:: m_numRowsX,m_numColumnsY zeroed so setMem always works
   setMem(A.size(), B.size());
-  for (size_t i = 0; i < nx; i++) {
-    for (size_t j = 0; j < ny; j++) {
-      V[i][j] = A[i] * B[j];
+  for (size_t i = 0; i < m_numRowsX; i++) {
+    for (size_t j = 0; j < m_numColumnsY; j++) {
+      m_rawData[i][j] = A[i] * B[j];
     }
   }
 }
 //
 template <typename T>
 Matrix<T>::Matrix(const std::vector<T> &data)
-    : nx(0), ny(0), V(nullptr) {
+    : m_numRowsX(0), m_numColumnsY(0) {
   size_t numel = data.size();
-  size_t nxt = static_cast<size_t>(sqrt(double(numel)));
-  size_t test = nxt * nxt;
+  size_t m_numRowsXt = static_cast<size_t>(sqrt(double(numel)));
```

Comment:
Oh woops - I've changed it so that `setmem` was a private method which takes no params to tidy it up.

To avoid this PR becoming too large I'll rename the variable names as suggested and hold onto the diff for a future PR?